loki.config.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
     "^design system",
     "^Inputs/DatePicker Dates range",
     "^Parameters/DatePicker",
+    "^Buttons/Button Compact size, custom color",
```

Comment:
add a single story. The story filter should be in this format `<story-structure>[<single-space><story-name>]` where the story name is optional